pattern_import_export/models/ir_exports.py

```diff
@@ -71,28 +68,9 @@ def _get_data_to_export(self, records):
         This function could be recursive in case of sub-pattern
         """
         self.ensure_one()
+        json_parser = self.export_fields._get_json_parser_for_pattern()
```

Comment:
explain me why you move this

Reply:
I moved some things around, one was to avoid calling get_parser for each record, and on the fly I changed this also. What I had in mind was probably to support this in the future: being able to define arbitrary columns inside the extra tabs, not just the key column